### ISSUE TITLE
Do custom github domains cost money?

### ISSUE THREAD

**Issue by TheRealDigitalMaster** [3kh0/3kh0.github.io#338]
I am making a gaming website (not branching this repo), and it is disguised as an educational websites to fool the teachers. Does custom domains cost money?

**Comment by RetroWonder** [comment]
No, you just have to edit settings, however this would be a question for the Discord

**Comment by TheRealDigitalMaster** [comment]
My discord is blocked

**Comment by 3kh0** [comment]
Use a proxy

**Comment by TheRealDigitalMaster** [comment]
Ok

**Comment by TheRealDigitalMaster** [comment]
Ok but i have discord now.

**Comment by TheRealDigitalMaster** [comment]
But no phone number ._. im a minor

**Comment by jbxskm** [comment]
when i used to use discord a year ago they kept forcing me to verify a phone number
im a minor too

**Comment by TheRealDigitalMaster** [comment]
ikr but is there a way to embed code into the website to bypass it? 🤔

**Comment by TheRealDigitalMaster** [comment]
*or embedding code into the app

**Comment by jbxskm** [comment]
WAT? HUNH? VATTT???

**Comment by TheRealDigitalMaster** [comment]
to bypass the ‘verify phone number’ thing

**Comment by RetroWonder** [comment]
> to bypass the ‘verify phone number’ thing

Yeah, it's called wait a few years & be 13

**Comment by TheRealDigitalMaster** [comment]
im over 13 but under 18 and i dont have a phone my parents said i cant until im 18

**Comment by 3kh0** [comment]
damn

**Comment by jbxskm** [comment]
oof
 i have to buy one with my own money if i want one
